src/seedu/addressbook/data/person/Person.java

```diff
@@ -83,5 +83,9 @@ public int hashCode() {
     public String toString() {
         return getAsTextShowAll();
     }
+    
+    public String toStringWithoutTags() {
```

Comment:
Good method naming =).
This could be improved with commenting of what this method does.
Again refer to the comment in T4A3.